### PR TITLE
new docs for OAuth Apps management page

### PR DESCRIPTION
**Pull request by rzyxie**
### What does this PR do?
Documentation for new OAuth Apps management page under Organization Settings 

org-management-page

### Motivation
GA for org OAuth Apps management page

### Preview

https://docs-staging.datadoghq.com/rebecca.xie/oauth-apps-management/account_management/org_settings/
https://docs-staging.datadoghq.com/rebecca.xie/oauth-apps-management/account_management/org_settings/oauth_apps/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/org_settings.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ The **Audit Logs** tab in the Organization Settings page opens a new tab to the
 
 The **Audit Log Settings** tab allows you to set the retention period of audit logs and enable archiving to other cloud storage services.
 
+#### OAuth Apps
+
+The [**OAuth Apps**][9] tab allows you to view or manage OAuth applications in your organization.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings.md:73]:
```suggestion
The [**OAuth Apps**][9] page allows you to view or manage OAuth applications in your organization. 
```

File: content/en/account_management/org_settings.md [hunk 2]
```diff
@@ -84,3 +88,4 @@ To rename your organization, click the **Rename organization** tab, enter the ne
 [6]: /account_management/api-app-keys/
 [7]: /account_management/api-app-keys/#client-tokens
 [8]: /events/guides/email/
+[9]: /account_management/oauth_apps
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings.md:91]: I pushed a commit to re-arrange the page:
```suggestion
[9]: /account_management/org_settings/oauth_apps
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+title: OAuth Apps
+kind: documentation
+further_reading:
+- link: "/account_management/org_settings/"
+  tag: "Documentation"
+  text: "Learn more about organization settings"
+---
+## Overview
+
+The **OAuth Apps** management page in [Organization Settings][1] helps you manage and gain visibility into your organization’s OAuth applications, such as the scopes and permissions granted to an application and the users that have authorized access for it. This includes the Datadog Mobile App and your custom [Datadog Apps][2] that have [OAuth API Access][3] enabled.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:11]:
```suggestion
Use the **OAuth Apps** management page under [Organization Settings][1] to manage and gain visibility into your organization's OAuth applications, such as the scopes and permissions granted to an application and the users that have authorized access for it.
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  tag: "Documentation"
+  text: "Learn more about organization settings"
+---
+## Overview
+
+The **OAuth Apps** management page in [Organization Settings][1] helps you manage and gain visibility into your organization’s OAuth applications, such as the scopes and permissions granted to an application and the users that have authorized access for it. This includes the Datadog Mobile App and your custom [Datadog Apps][2] that have [OAuth API Access][3] enabled. 
+
+{{< img src="account_management/oauth_apps/org-management-page.png" alt="OAuth Apps management page in Datadog" style="width:100%;">}}
+
+## Permissions
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:15]:
```suggestion
## Setup
### Permissions
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+By default, users with [Datadog Standard and Datadog Admin roles][4] can access the OAuth Apps management page. If your organization has [custom roles][5] defined, add your user to any custom role with `org_authorized_apps_read` and `org_authorized_apps_write` permissions. 
+
+Only users with the Datadog Admin role or the `org_authorized_apps_write` permission can manage OAuth applications on this page, such as disabling applications or revoking OAuth access for a user.
+
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:20]:
```suggestion

### Enable

OAuth applications include the Datadog Mobile App and your custom [Datadog Apps][2] that have [OAuth API Access][3]. 
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+## Disable an OAuth application
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:21]:
```suggestion
### Disable
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+Disabling OAuth access for an application revokes access to this application for all users in your organization. While the application remains installed, users are no longer able to use the application and will be prompted with an error if they attempt to authorize it.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:23]:
```suggestion
Disabling OAuth access for an application revokes access to this application for all users in your organization. While the application remains installed, users are no longer able to use the application and are prompted with an error if they attempt to authorize it.
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+There are two ways to disable an application from the OAuth Apps management page:
+1. Hover over your application in the apps table to reveal the **Disable** button on the right-hand side of the row.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:26]:
```suggestion
1. Hover over your application in the apps table to reveal the **Disable** button on the right side of the row.
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+{{< img src="account_management/oauth_apps/disable-app-table.png" alt="Disable button in apps table" style="width:100%;">}}
+
+2. Click on your application to open the detailed view of the application and reveal the **Disable Application** button.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:29]:
```suggestion
2. Click on your application to open the detailed view of the application and click the **Disable Application** button.
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+{{< img src="account_management/oauth_apps/disable-app-detailed.png" alt="Disable button in app's detailed view" style="width:100%;">}}
+
+**Note**: Enabled applications allow users with necessary permissions to grant it access on their behalf. When re-enabled, users that have previously authorized the application are required to re-authorize the application to gain access again.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:32]:
```suggestion
**Note**: Enabled applications allow users with necessary permissions to grant it access on their behalf. When re-enabled, users that previously authorized the application are required to re-authorize the application to regain access.
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+## Revoke OAuth access for a user
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:34]:
```suggestion
### Revoke access
```

File: content/en/account_management/org_settings/oauth_apps.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+
+Revoking OAuth access to an application removes all access that a user has to the application. If the user has the required permissions to authorize the application, they can gain access again by re-authorizing it.
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:36]:
```suggestion
Revoking a user's OAuth access to an application removes all access to that application. If the user has the required permissions to authorize the application, they can regain access by re-authorizing it.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

File: content/en/account_management/org_settings/oauth_apps.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+## Setup
+### Permissions
+
+By default, users with [Datadog Standard and Datadog Admin roles][4] can access the OAuth Apps management page. If your organization has [custom roles][5] defined, add your user to any custom role with `org_authorized_apps_read` and `org_authorized_apps_write` permissions. 
+
+Only users with the Datadog Admin role or the `org_authorized_apps_write` permission can manage OAuth applications on this page, such as disabling applications or revoking OAuth access for a user.
+
+### Enable
+
+Enabled OAuth applications allow users with necessary permissions to authorize access on their behalf.  OAuth applications include the Datadog Mobile App and your custom [Datadog Apps][2] that have [OAuth API Access][3].
```
Comment (ruthnaebeck) [inline content/en/account_management/org_settings/oauth_apps.md:24]:
```suggestion
Enabled OAuth applications allow users with necessary permissions to authorize access on their behalf. OAuth applications include the Datadog Mobile App and your custom [Datadog Apps][2] that have [OAuth API Access][3]. 
```

**Review by ruthnaebeck — COMMENTED** [review]
👍

**Review by ruthnaebeck — APPROVED** [review]
👍

**Review by cswatt — APPROVED** [review]

**Pull request merged**